adserver/templates/adserver/email/publisher-payout.html

```diff
@@ -1,9 +1,20 @@
 {% autoescape off %}
 
+{# TODO: Find a way to make this non-EA specific,
+but for now I think we can't avoid it because of linking to content on our marketing pages. #}
```

Comment:
I'm pretty sure these types of comments are *single line* comments. If you want multi-line, you need to do `{% comment %}`